README.md

```diff
@@ -25,7 +25,9 @@ If you run into any issues while working in this repo, please check out [the tro
 
 `npm` scripts are provided for common tasks. These include:
 
-- `npm run start` - runs the UI in development mode
+- `npm run test` - runs all tests for the client and server
+- `npm run start` - runs the UI client and server in development mode
+  - `npm run addDevCerts` - requires `openssl`, will generate certificates for development purposes for UI development
```

Comment:
I err away from checking in certs, hence having them generated/in a directory we ignore. Will add openssl to Getting started - I wanted it to be an option, rather than a requirement